packages/web-react/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ElementType } from 'react';
+import classNames from 'classnames';
+import { useBreadcrumbsStyleProps } from './useBreadcrumbsStyleProps';
+import { SpiritBreadcrumbsProps } from '../../types';
+import { useStyleProps } from '../../hooks/styleProps';
+
+const defaultProps = {
+  items: [],
+};
+
+export const Breadcrumbs = <T extends ElementType = 'nav'>(props: SpiritBreadcrumbsProps<T>): JSX.Element => {
+  const { elementType: ElementTag = 'nav', ...restProps } = props;
+  const { classProps, props: modifiedProps } = useBreadcrumbsStyleProps(restProps);
+  const { styleProps, props: otherProps } = useStyleProps(modifiedProps);
+
+  return (
+    <ElementTag
+      {...otherProps}
+      {...styleProps}
+      className={classNames(classProps, styleProps.className)}
+      aria-label="Breadcrumb"
+    >
+      <ol>
+        {restProps.items.map((item, index) => (
```

Comment:
I think it would be better to destructure `restProps` and use only `items`.

Reply:
I tried to destructure it but `useBreadcrumbsStyleProps` expects the `SpiritBreadcrumbsProps` (same as other components do).

I think I need some help here, please.

![Screenshot 2022-08-22 at 13 59 35](https://user-images.githubusercontent.com/9672439/185919542-e4beb16d-2f0b-4a76-8a8e-1ed6781edc19.png)

Reply:
I this case the `useBreadcrumbsStyleProps` should expect only `BreadcrumbsProps`, because you destructure them in the component `const { elementType: ElementTag = 'nav', items, goBackItemTranslation, ...restProps } = props;`. You do not want to let the `items` and `goBackItemTranslation` go to `{...otherProps}` because they are not valid HTML attributes.